Zune.DB.Console/Program.cs

```diff
@@ -15,7 +15,7 @@ class Program
         static async Task Main(string[] args)
         {
             // Set up CLI options
-            string connectionString = "mongodb://localhost:27017";
+            string connectionString = "mongodb://mongo:27017";
```

Comment:
This should be pulled into `IConfiguration` so that it can be one of many configuration types including environment variables.